src/components/Dashboard/menuItems.js

```diff
@@ -7,14 +7,6 @@ export default [
     value: 'Releases',
     path: '/releases',
   },
-  {
-    value: 'Permissions',
-    path: '/permissions',
-  },
-  {
-    value: 'Signoffs',
-    path: '/signoffs',
-  },
```

Comment:
Nitpicking - feel free to ignore this comment

I think we can group all of the links here:

```jsx
export default {
  main: [
    {
      value: 'Rules',
      path: '/rules',
    },
    {
      value: 'Releases',
      path: '/releases',
    },
    {
      value: 'History',
      path: '/history',
    },
  ],
  settings: [
    {
      value: 'Users',
      path: '/users',
    },
    {
      value: 'Roles',
      path: '/roles',
    },
    {
      value: 'Required Signoffs',
      path: '/required-signoffs',
    },
  ],
};
```

Reply:
Good idea, I'll do it.